tests/test_forcing_nudging.py

```diff
@@ -0,0 +1,153 @@
+"""
+Tests for checking different options related to nudging when doing forcing
+conversion
+"""
+import uuid
+from pathlib import Path
+
+import pytest
+import xarray as xr
+import yaml
+
+import lagtraj
+from lagtraj.forcings import ForcingLevelsDefinition, ForcingSamplingDefinition
+
+FORCING_CONVERSION_BASE = """
+levels_method   : copy
+export_format   : kpt
+comment         : Forcing and initial conditions for Lagrangian case
+campaign        : EUREC4A
+source_domain   : n/a
+reference       : n/a
+author          : s.j. boeing
+modifications   : First version
+case            : n/a
+adv_temp        : 1
+adv_theta       : 1
+adv_thetal      : 1
+adv_qv          : 1
+adv_qt          : 1
+adv_rv          : 1
+adv_rt          : 1
+rad_temp        : 0
+rad_theta       : 0
+rad_thetal      : 0
+forc_omega      : 0
+forc_w          : 1
+forc_geo        : 1
+surfaceType     : ocean
+surfaceForcing  : ts
+surfaceForcingWind               : z0_traj
+"""
+
+
+VALID_YAML_EXAMPLES = [
+    """
+nudging_method_scalars: constant
+nudging_timescale_scalars: 10
+""",
+    """
+nudging_method_scalars: runtime_inversion_height
+nudging_timescale_scalars: 10
+nudging_layer_thickness_scalars: 3000.0
+nudging_shape_scalars: cos
+""",
+    """
+nudging_method_scalars: fixed_height
+nudging_timescale_scalars: 10
+nudging_layer_thickness_scalars: 3000.0
```

Comment:
I suggest changing "layer_thickness" to "transition_thickness", so it is explicit. Not sure if "height_above"/"lowest_height" is better for height.